src/scene/scene.js

```diff
@@ -150,6 +150,13 @@ class Scene extends EventHandler {
      */
     sky = null;
 
+    /**
+     * Use physically based camera parameters to describe exposure
+     *
+     * @type {boolean}
+     */
+    usePhysicallyBasedCamera = false;
```

Comment:
shouldn't this be a camera setting?

Reply:
we discussed that .. if you have multiple cameras and multiple lights, they probably all should be in the either physical or not .. so a global switch perhaps makes sense?

Reply:
even though it could perhaps be renamed to 'physicalBasedIntensities' or something

Reply:
I pushed this change.

Reply:
So lights will also have legacy and physically-based units?

Reply:
Yeah, we already expose the exposure setting on the camera, and that 'fits' with the old light intensities. These units will have to be enabled in parallel for it to be consistent.

Reply:
Just a little note to say this property is pretty verbose. `physicalCameras`?

Reply:
It's usePhysicallyBasedUnits currently, we could call it physicalUnits I guess.